docs/program/apis/session.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Monitor resource sessions with the Session API"
+linkTitle: "Session API"
+weight: 20
+type: "docs"
+description: "Safety monitor resources and attach contexts to sessions with the Session API."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server`.
+The period of time during which these clients are present is called a *session*.
+
+## API
+
+The `Session` API supports the following methods:
```

Comment:
Question for @dgottlieb or any SME who has a better idea of what's going on in the RDK than I do: is this correct? Or should this all be replaced with https://pkg.go.dev/go.viam.com/rdk@v0.8.0/session#Session as these are just the methods the `session` package offers and the interface for the type itself has different methods (Active, Heartbeat, etc.). Should both be on here? Apologies for the confusion

Reply:
Similarly, for posterity, these are internal data types that are not client facing.